trade-finance-manager-ui/templates/_macros/header.njk

```diff
@@ -1,56 +1,27 @@
 {% from "moj/components/header/macro.njk" import mojHeader %}
-
 {% macro render(params) %}
-
-  {% if params.user %}
-
-    {{ mojHeader({
-      organisationLabel: {
-        text: 'UK Export Finance',
-        href: '#'
-      },
-      serviceLabel: {
-        text: 'Trade Finance Manager',
-        href: '#'
-      },
-      navigation: {
-        label: 'Account navigation',
-        items: [
-          {
-            text: params.user.firstName + ' ' + params.user.lastName,
-            href: '#',
-            attributes: {
-              'data-cy': 'header-user-link'
-            }
-          },
-          {
-            text: 'Sign out',
-            href: '/logout',
-            attributes: {
-              'data-cy': 'header-sign-out-link'
-            }
-          }
-        ]
-      }
-    }) }}
-
-  {% else %}
-
-    {{ mojHeader({
-      organisationLabel: {
-        text: 'UK Export Finance',
-        href: '#'
-      },
-      serviceLabel: {
-        text: 'Trade Finance Manager',
-        href: '#'
-      }
-    }) }}
-
-  {% endif %}
-
-
-  {% include "_partials/primary-navigation.njk" %}
-
-
-{% endmacro %}
+  <header class="moj-header" role="banner">
+    <div class="moj-header__container">
+      <div class="moj-header__logo">
+        <img
+          src="/assets/images/UK-Export-Finance-logo.svg"
```

Comment:
Missing `alt` and `title` tag from `<img>`.

Reply:
done